### PR TITLE
Updates made to readme.md

### PR DESCRIPTION
**Pull request by sandeepchads**
Updated readme.md with the migration steps. Yet to update:
1) Location of the EXE
2) Location of the blog with updates to the Wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+
+## Detailed steps
+Here are the detailed steps for Wiki migration:
+
+1.	Goto Wiki* hub in VSTS.  Create your first Wiki page (You can create a dummy page and always delete it later)
+ 
+2.	Click on More -> Clone Wiki -> and clone your VSTS Wiki repo using 'git clone' command.
+
+<img src="https://github.com/sandeepchads/vsts-wikiTools/blob/master/Images/1%20Clone%20wiki.PNG">
+[
```
Comment (sandeepchads) [inline README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+
+<img src="https://github.com/sandeepchads/vsts-wikiTools/blob/master/Images/2%20Git%20Clone.PNG">
+
+ 
+3.	Clone the Wiki extension repo. The Wiki will be mapped to a folder given by you during the wiki creation. You can check that by going to "manage wiki" option in the existing wiki as shown below.
+
+<img src="https://github.com/sandeepchads/vsts-wikiTools/blob/master/Images/3%20Wiki%20extension.PNG">
+
+The value under the label "Root" is the folder in your repo inside which the existing wiki pages are saved.
+4.	Say you have cloned the above mentioned "sampleWiki" in the location "C:\wiki\sampleWiki". The wiki pages will be saved in the path "C:\wiki\sampleWiki\_extensionWiki". Let this be "LocationB" with respect to the document.
```
Comment (sandeepchads) [inline README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+ 
+5.	Create an empty folder in any path of choice in your location machine and let that be "LocationC" with respect to the document.
+ 
+> Just to summarize 
+- Location A = VSTS Wiki 
+- Location B = Wiki extension 
+- Location C = Empty folder where we will run our migration tool'''
```
Comment (sandeepchads) [inline README.md]: Fixed both

**Comment by snankani** [comment]
this repository is in maintenance and no active is happening, hence we will be archiving this repo.

**Pull request closed without merging**